mmv1/third_party/terraform/services/apphub/data_source_apphub_discovered_workload_test.go

```diff
@@ -0,0 +1,113 @@
+package apphub_test
+
+import (
+    "testing"
+
+    "github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+    "github.com/hashicorp/terraform-provider-google/google/acctest"
+)
+
+func TestDataSourceApphubDiscoveredWorkload_basic(t *testing.T) {
+    t.Parallel()
+
+    context := map[string]interface{}{
+    	"random_suffix": acctest.RandString(t, 10),
+    }
+
+    acctest.VcrTest(t, resource.TestCase{
+        PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+        ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
+        Steps: []resource.TestStep{
+            {
+                Config: testDataSourceApphubDiscoveredWorkload_basic(context),
+            },
+        },
+    })
+}
+
+func testDataSourceApphubDiscoveredWorkload_basic(context map[string]interface{}) string {
+    return acctest.Nprintf(`
+resource "google_project" "service_project" {
+	project_id ="<%= ctx[:vars]['service_project_attachment_id'] %>"
+	name = "Service Project"
+	org_id = "<%= ctx[:test_env_vars]['org_id'] %>"
+}
+
+resource "google_apphub_service_project_attachment" "service_project_attachment" {
```

Comment:
This resource is not used anywhere and can be deleted.

Reply:
It is used currently.